code/game/gamemodes/dynamic/dynamic_midround_rolling.dm

```diff
@@ -101,3 +108,25 @@
 	var/heavy_coefficient = CLAMP01((next_midround_roll - midround_light_upper_bound) / (midround_heavy_lower_bound - midround_light_upper_bound))
 
 	return 100 * (heavy_coefficient * max(1, chance_modifier))
+
+/// Checks to see if all roundstart 'high impact' rulesets are dead, and tries to force an injection if so.
+/datum/game_mode/dynamic/proc/check_for_dead_high_impacts()
```

Comment:
Why does one have to be forced? Rather just allowing the injections when the next injection naturally occurs seems simpler and equally effective.

Yes it may add some delay but it works better within the system of dynamic and still allows for properly spaced / scaled injections without over complicating things by forcing an injection, which could potentially occur when one was about to naturally occur.

I'm not entirely against this as I do like that it responds promptly to the ruleset dying, but I don't like that it adds a factor of predictability to the injection, meaning you can kill off the last of a cultist and then boom something spawns within the next 5 minutes.

Reply:
I've removed the forced injection entirely for now, as I can't think of a good way to implement it.

Reply:
You could potentially reduce the time to the next injection or add a bit of threat refund